en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -431,30 +431,17 @@ A good tuning is obtained as follows:
 
 ### External Vision System
```

Comment:
THanks @bresch - much appreciated. I have rewritten this section. Are you happy to merge this PR?

Reply:
Yes, looks good now.